README.md

```diff
@@ -70,6 +70,34 @@ in your browser to access the preview.
 
 **Note:** The build of the site will take several minutes.
 
+### Preview with Docker
+
+Run the following
+
+``` sh
+docker run -it --name ruby-lang -v $PWD:/www.ruby-lang.org -w /www.ruby-lang.org --network host ruby:2.6.1 sh -c "bundle install --without production && bundle exec rake serve"
+```
+
+to generate the website and start a local web server
+
+
+Open [http://localhost:9292/](http://localhost:9292/)
```

Comment:
You can use `<http://localhost:9292/>` instead of `[http://localhost:9292/](http://localhost:9292/)`.